docs/api-testing/integrations/apifctl-cicd-integration.md

```diff
@@ -39,10 +39,6 @@ To generate a webhook:
 
 You can then reuse this Webhook for future tests within that Project by returning to the **Webhooks** tab and copying it there. Webhooks are Project-specific.
 
-:::info
```

Comment:
We have doc-specific formatting rules for notes. I will rewrite this to match our style and commit as a separate branch.

Reply:
@lauramthomas I didn't change any formatting rule, just removed some wrong statements. So the formatting is the same as it was previously